src/SdnDiagnostics.psd1

```diff
@@ -92,18 +97,27 @@
         'Get-SdnVfpPortLayer',
         'Get-SdnVfpPortRule',
         'Get-SdnVfpPortState',
+        'Import-SdnCertificate',
         'Install-SdnDiagnostics',
         'Invoke-SdnCommand',
         'Invoke-SdnGetNetView',
         'Invoke-SdnServiceFabricCommand',
         'Move-SdnServiceFabricReplica',
+        'New-SdnCertificate',
+        'New-SdnCertificateRotationConfig',
+        'New-SdnNetworkControllerCertificate',
+        'Set-SdnCertificateAcl',
+        'Set-SdnServiceFabricClusterConfig',
         'Set-SdnVMNetworkAdapterPortProfile',
         'Show-SdnVfpPortConfig',
+        'Start-SdnCertificateRotation',
         'Start-SdnDataCollection',
         'Start-SdnEtwTraceCapture',
         'Start-SdnNetshTrace',
+        'Start-SdnNetworkControllerCertificateUpdate',
```

Comment:
Do we need this exposed anymore since our primary function entry point is `Start-SdnCertificateRotation` ?